src/data_snack/connections/base.py

```diff
@@ -19,16 +19,26 @@ def get(self, key: Text) -> bytes:
         """
         ...
 
-    def set(self, key: Text, value: Union[Text, bytes]) -> bool:
+    def set(self, key: Text, value: Union[Text, bytes], expire: int = 0) -> bool:
         """
         Saves given value using provided key.
 
         :param key: unique data identifier
         :param value: value saved in db
+        :param expire: optional int, number of seconds until the item is expired, or zero for no expiry
```

Comment:
Yeah your right, there is no need to say 'optional int' here. 

But it is optional argument tho, `Optional` typing is something different _"Note that this is not the same concept as an optional argument, which is one that has a default. An optional argument with a default does not require the Optional qualifier on its type annotation just because it is optional."_

Reply:
Yeah, but we what is the reason to specify it in docstring?

Reply:
Like I said above, your right that there is no need to write `optional int` here. I will change it `number of seconds until the item is expired, or zero for no expiry`

Reply:
Done